notebooks/00_quick_start/sar_aml_compute_movielens.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>in the other notebooks we are adding the imports in the first cell</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Moved imports to first cell